src/contextify.cc

```diff
@@ -126,27 +128,28 @@ class ContextifyContext : public ObjectWrap {
 class ContextWrap : public ObjectWrap {
 public:
     static void Init(void) {
-        NanScope();
-        Local<FunctionTemplate> tmpl = NanNew<FunctionTemplate>();
+        Nan::HandleScope scope;
+
+        Local<FunctionTemplate> tmpl = Nan::New<FunctionTemplate>();
         tmpl->InstanceTemplate()->SetInternalFieldCount(1);
-        NanAssignPersistent(functionTemplate, tmpl);
-        NanAssignPersistent(constructor, tmpl->GetFunction());
+        functionTemplate.Reset(tmpl);
+        constructor.Reset(tmpl->GetFunction());
     }
 
     static Local<Context> createV8Context(Handle<Object> jsContextify) {
-        NanEscapableScope();
-        Local<Object> wrapper = NanNew(constructor)->NewInstance();
+        Nan::EscapableHandleScope scope;
+        Local<Object> wrapper = Nan::New(constructor)->NewInstance();
 
         ContextWrap *contextWrapper = new ContextWrap();
         contextWrapper->Wrap(wrapper);
 
-        Local<Object> obj = NanNew(jsContextify);
-        NanMakeWeakPersistent(obj, contextWrapper, &weakCallback);
+        Local<Object> obj = Local<Object>::New(Isolate::GetCurrent(), jsContextify);
```

Comment:
Ah, I guess `jsContextify` is a `Persistent<Object>`. At some point I should see if there's a good way of making the explicit template parameter work in such cases. Using the overload `Nan::New(jsContextify);` should produce the intended result.